events.lisp

```diff
@@ -577,7 +577,9 @@ the window in it's frame."
       (group-focus-window (window-group win) win))))
 
 (define-stump-event-handler :enter-notify (window mode)
-  (when (and window (eq mode :normal) (eq *mouse-focus-policy* :sloppy))
+  (when (and window (eq mode :normal)
+             (and (eq *mouse-focus-policy* :sloppy)
+                  (not *sloppy-mouse-focus-ignored*)))
```

Comment:
Instead of this global why not change the `*mouse-policy-focus*` to ignore and then restore the previous policy after the mouse has been re-positioned?